packages/venia-ui/lib/components/AccountMenu/accountMenu.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { bool, shape, string } from 'prop-types';
+
+import { mergeClasses } from '@magento/venia-ui/lib/classify';
+
+import AccountMenuItems from './accountMenuItems';
+import defaultClasses from './accountMenu.css';
+
+const AccountMenu = React.forwardRef((props, ref) => {
```

Comment:
What's the difference between using `React.forwardRef` and just passing a custom prop like `asideRef`

Reply:
I think it would still work using a `prop` honestly. I didn't actually try it though because when I went to the docs for `useRef` I noticed `forwardRef`.

It lets you add a `ref` to your custom components like `<MyComponent ref={myRef} />`, and then internally `MyComponent` can assign `myRef` to an actual DOM element. I think that's probably just a nice to have though.